qiskit_bot.yaml

```diff
@@ -445,6 +445,7 @@ notifications:
     - "@jenglick"
     - "@garrison"
     - "@johannesgreiner"
+    - "@ElePT"
```

Comment:
This one is not a tutorial, is that intentional?

Reply:
Yes, I updated based on feedback from @pandasa123